includes/admin/class-sensei-admin-notices.php

```diff
@@ -241,11 +241,20 @@ private function add_admin_notice( $notice_id, $notice ) {
 			$notice_class = 'sensei-notice-' . $notice['style'];
 		}
 
-		wp_enqueue_script( 'sensei-dismiss-notices' );
-
+		$is_dismissible       = $notice['dismissible'];
+		$notice_wrapper_extra = '';
+		if ( $is_dismissible ) {
+			wp_enqueue_script( 'sensei-dismiss-notices' );
```

Comment:
This is not changing from before but: is it ok to `wp_enqueue_script` the same script multiple times?

Reply:
Yeah, it doesn't have any effect.